platform-ajax/src/test/java/com/softicar/platform/ajax/input/auto/complete/AbstractAjaxAutoCompleteTest.java

```diff
@@ -144,23 +144,25 @@ private void assertContainsText(String string, String subString) {
 
 	public static enum Indicator implements IAjaxSeleniumTestDomAutoCompleteInputIndicator {
 
+		// TODO remove COMMITTING and LOADING
+		// TODO use DomAutoCompleteIndicatorType directly
 		COMMITTING("committing", "AutoCompleteIndicatorCommitting"),
```

Comment:
second arguments in this enum should use enumerated CSS classes, instead of plain strings

Reply:
removed